frontend/src/v5/ui/routes/viewer/tickets/ticketsList/ticketItem/ticketItem.component.tsx

```diff
@@ -46,6 +50,14 @@ export const TicketItem = ({ ticket, onClick, selected }: TicketItemProps) => {
 	} = ticket;
 	const riskLevel = ticket.modules?.safetibase?.['Level of Risk'];
 	const treatmentStatus = ticket.modules?.safetibase?.['Treatment Status'];
+	const updateTicketProperty = (value) => TicketsActionsDispatchers
+		.updateTicket(teamspace, project, containerOrFederation, ticket._id, { properties: value }, isFederation);
+	const onBlurAssignees = (newVals) => {
+		if (!isEqual(newVals, assignees)) updateTicketProperty({ Assignees: newVals });
+	};
+	const onBlurDueDate = (newVal) => {
+		if (newVal !== dueDate) updateTicketProperty({ 'Due Date': newVal });
```

Comment:
I think the issues propertie names should be in a const (or enum) to be shared between the files instead of hardcoding it like this. 
That way when we use the same const when we make the specialized version for the ticket details..
Same thing with safetibase properties.

Reply:
I have created an enum for this